app/views/dashboard/layouts/_main.html.erb

```diff
@@ -2,6 +2,13 @@
 
   <!-- Top Menu -->
   <header class="relative w-full bg-white">
+      <% if @map&.sync_mode %>
+        <div class="flex flex-row items-center justify-center w-full bg-mapzy-red text-white">
+          <p>
+            You have turned Sync Mode on. In this mode, you can't manage your map through the dashboard.
+          </p>
```

Comment:
Let's also add where to go to disable it? In case it was done by mistake or forgotten.

Sync Mode is on. You cannot manage your map through the dashboard. Go to the settings to disable it.

Reply:
Fixed in 86d8dce